### PR TITLE
Fix double "use strict"

### PR DESCRIPTION
**Pull request by jhnns**
**What kind of change does this PR introduce?**

Sort of bug fix

**Did you add tests for your changes?**

No

**Summary**

This fixes a problem where webpack rendered modules with two "use strict" statements at the beginning

**Does this PR introduce a breaking change?**

No

### CODE REVIEW

File: test/statsCases/tree-shaking/expected.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Hash: 9e57dbeb6deda98eb9bf
 Time: Xms
     Asset     Size  Chunks             Chunk Names
-bundle.js  7.08 kB       0  [emitted]  main
+bundle.js  7.07 kB       0  [emitted]  main
```
Comment (jhnns) [inline test/statsCases/tree-shaking/expected.txt]: Yay, our bundle is 0.01 kB smaller now 😉

Comment (sokra) [inline test/statsCases/tree-shaking/expected.txt]: Awesome

File: lib/FunctionModuleTemplatePlugin.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ FunctionModuleTemplatePlugin.prototype.apply = function(moduleTemplate) {
 			defaultArguments.push("__webpack_require__");
 		}
 		source.add("/***/ function(" + defaultArguments.concat(module.arguments || []).join(", ") + ") {\n\n");
-		if(module.strict) source.add("\"use strict\";\n");
+		if(module.strict) source.add("\"use strict\";");
```
Comment (sokra) [inline lib/FunctionModuleTemplatePlugin.js]: Why removing this new line?

**Comment by rohmanhm** [comment]
Merge it, I got double 'use strict' too in my `bundle.js` file

```
/* 1 */
/***/ function(module, exports, __webpack_require__) {

"use strict";
'use strict';

var _blabla = __webpack_require__(0);

var _blabla2 = _interopRequireDefault(_blabla);
```

**Comment by sokra** [comment]
Could you add a test that makes sure that only one `use strict` is in the bundle?

**Comment by jhnns** [comment]
Haha, yes, you got me. I actually thought about that, but I haven't found a place where to put the test. Where would you put it?

**Comment by sokra** [comment]
Tests can read the bundle file like here: https://github.com/webpack/webpack/blob/54aa3cd0d6167943713491fd5e1110b777336be6/test/configCases/target/umd-named-define/index.js

`test/cases/code-generation/...`

**Pull request merged**